cdk.json

```diff
@@ -37,17 +37,28 @@
       "kustomizations": [
         {
           "kustomizationPath": "./artifacts/grafana-operator-manifests/eks/infrastructure"
+        },
```

Comment:
Remove all neuron references from `cdk.json`. `cdk.json` is placeholder for generic pattern.